charts/xp-management/values.yaml

```diff
@@ -165,6 +165,10 @@ externalPostgresql:
   password: password
   # -- Host address for the External postgres
   address: 127.0.0.1
+  # -- Secret name which contains credentials to access externalPostgresql
+  secretName: ""
```

Comment:
I think there is one other flag here in other charts to enable/disable secret creation. Can you check and make it consistent?

Reply:
What do you mean?

Reply:
eg: https://github.com/caraml-dev/helm-charts/blob/main/charts/caraml-authz/values.yaml#L50 and https://github.com/caraml-dev/helm-charts/blob/main/charts/caraml-authz/templates/database-secret.yaml

You will need to create the secret for cases when you are using an external Postgres and the secret is not created by others.

Reply:
Added.